frontend/__tests__/src/client/MetricsClient.test.ts

```diff
@@ -47,4 +49,12 @@ describe('get steps from metrics response', () => {
       await metricsClient.getSteps(params, buildId, organizationId, pipelineType, token)
     }).rejects.toThrow(VERIFY_ERROR_MESSAGE.BAD_REQUEST)
   })
+
+  it('should show isNoStep True when getSteps response status 204', async () => {
```

Comment:
1. should set isNoStep to true when ....
maybe this is better than yours

2. haveStep is better than isNoStep

Reply:
Done